webui/src/pages/Settings/Settings.tsx

```diff
@@ -26,51 +28,58 @@ const Settings = () => {
     });
   };
 
+  const jobsOptions: { label: string; key: keyof TableSettings }[] = [
+    { label: "Job ID", key: "showJobId" },
+    { label: "Name", key: "showJobName" },
+    { label: "Created", key: "showCreated" },
+    { label: "Program", key: "showProgram" },
+    { label: "Job Type", key: "showJobType" },
+    { label: "Label", key: "showLabel" },
+    { label: "Status", key: "showStatus" },
+  ];
```

Comment:
Can we use a similar technique to avoid hard-coding the `TableSettingsContext` i.e. not have to list each config as a separate variable?